view/src/main/java/cmm/apps/esmorga/view/MainActivity.kt

```diff
@@ -45,9 +62,51 @@ class MainActivity : ComponentActivity() {
 
     private fun setupNavigation(loggedIn: Boolean) {
         setContent {
-            val navigationController = rememberNavController()
-            EsmorgaNavigationGraph(navigationController = navigationController, loggedIn = loggedIn)
+            EsmorgaTheme {
+                val navigationController = rememberNavController()
+                val bottomNavItems = listOf(
+                    BottomNavItem.Explore,
+                    BottomNavItem.MyEvents,
+                    BottomNavItem.Profile
+                )
+
+                HomeView(bottomNavItems, navigationController) {
+                    EsmorgaNavigationGraph(navigationController = navigationController, loggedIn)
+                }
+            }
+        }
+    }
+}
+
+@SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
```

Comment:
this annotation is weird

Reply:
is to avoid the error message for not using the scaffold innerpadding